internal/controllers/clusterctl/config.yaml

```diff
@@ -11,18 +11,18 @@ data:
     providers:
     # Cluster API core provider
     - name:         "cluster-api"
-      url:          "https://github.com/rancher-sandbox/cluster-api/releases/v1.7.7/core-components.yaml"
+      url:          "https://github.com/rancher-sandbox/cluster-api/releases/v1.8.4/core-components.yaml"
```

Comment:
Q: updatecli bumps it to the whatever latest version we release in the fork (v1.8.4 in this case), however, when I check https://github.com/rancher-sandbox/cluster-api/releases/tag/v1.8.4 it does not normally include all manifests unless it is re-released after manually running GH workflow https://github.com/rancher/clusterapi-forks/actions/workflows/core.yaml. So, does that mean, we should always verify all release manifests do exists for specific version of project bumped with updatecli PR prior to merging it?

Reply:
This is a good point. We could also use updatecli's [conditions](https://www.updatecli.io/docs/core/condition/), which would let us add a validation step before applying changes and opening the PR. It can be used to check for the existence of a file in a release. Perhaps we can take a look at which files we should be looking at before bumping versions.

Reply:
Files vary depending on if it is an infra provider/core CAPI/bootstrap/controlplane. The only file that all of them should have is `metadata.yaml`

Reply:
Or if files have *-components.yaml* in it, that should also work